### PR TITLE
(fix): Clicking Sign up button in navbar landing to Sign in page

### PR DESCRIPTION
**Pull request by subhadeepbanerjee2003**
`screenshot`
Screenshot from 2023-10-04 16-47-53

fixes : #23 

### Summary by CodeRabbit

**Release Notes:**

- New Feature: Updated user authentication flow. Users will now be redirected to the "/join" page for signing up and the "/login" page for signing in.
- Chore: Added new environment variables `NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY` and `CLERK_SECRET_KEY` for managing Clerk services.

### CODE REVIEW

**Review by Swpn0neel — CHANGES_REQUESTED** [review]
Sorry to say, but I think you haven't understood the issue properly.

You don't have to change the "Sign UP" text to "Sign In" text, you have to change the route so that the new users are redirected to the sign up page, not the sign in page.

File: .env.local
Comment (Arindam200) [inline .env.local:1]: Hey, @subhadeepbanerjee2003 ! You have mistakenly pushed the API keys! Please Delete that!

File: src/app/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ export default function App({ featureRef }) {
             as={Link}
             href="/login"
```
Comment (Arindam200) [inline src/app/components/Navbar/Navbar.jsx:85]: Hey! @subhadeepbanerjee2003 ! Thanks for the Contribution.

All you have to do is to change the href from "/login" to "/join"
```suggestion
            href="/join"
```

File: src/app/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -97,7 +97,7 @@ export default function App({ featureRef }) {
...
```
Comment (Arindam200) [inline src/app/components/Navbar/Navbar.jsx:98]:
```suggestion
            href="/join"
```

Same Here ! Change the href!

File: src/app/components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -97,7 +97,7 @@ export default function App({ featureRef }) {
...
           >
-            Sign Up
+            Sign In
```
Comment (Arindam200) [inline src/app/components/Navbar/Navbar.jsx:100]:
```suggestion
            Sign Up
```

File: src/app/components/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -84,7 +84,7 @@ export default function App({ featureRef }) {
...
```
Comment (Arindam200) [inline src/app/components/Navbar/Navbar.jsx:87]:
```suggestion
            Sign up
```

**Comment by Arindam200** [comment]
@subhadeepbanerjee2003  Please delete the `.env.local` file

**Review by Arindam200 — APPROVED** [review]

**Comment by Arindam200** [comment]
Thanks  @subhadeepbanerjee2003  for contributing! Keep Contributing!

**Comment by subhadeepbanerjee2003** [comment]
@Arindam200 thanks for guiding

**Pull request merged**